src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,17 @@
 
 public class MainApp {
 
+    public static void main() {
+
+        Machine excavator = new Excavator();
+        Machine bulldozer = new Bulldozer();
+        Machine truck = new Truck();
```

Comment:
The extra variables don't needed. You can create a new classes instances right inside `new Machine[] {new Excavarator(), ...}` block